mongo/mongolog/mongologger.go

```diff
@@ -46,7 +46,7 @@ func NewMongoLogger(opts ...*Options) (*MongoLogger, error) {
 			dl.outputFile = *mlo.OutputFile
 			dl.writer, err = os.Create(dl.outputFile)
 			if err != nil {
-				return nil, err
+				return nil, fmt.Errorf("Error opening logging output file: %v", err)
```

Comment:
User-facing errors should start with lowercase letters (see https://github.com/golang/go/wiki/CodeReviewComments#error-strings).